db/src/urls.rs

```diff
@@ -52,3 +62,48 @@ pub async fn add_url(
         ))
     }
 }
+
+pub async fn fetch_url_from_pages_table(pool: &PgPool, page_id: Uuid) -> color_eyre::Result<Page> {
```

Comment:
Ahh this function is un-used now right? Might just remove it in that case, I think its simple enough to write again if we need it later!